core/src/models/Destination.ts

```diff
@@ -740,4 +740,12 @@ export class Destination extends LoggedModel<Destination> {
       where: { destinationId: instance.id },
     });
   }
+
+  @AfterDestroy
+  static async unassociateRelatedExports(instance: Destination) {
+    return Export.update(
+      { destinationId: null },
+      { where: { destinationId: instance.id } }
+    );
+  }
 }
```

Comment:
👍 this is great as a hook on the destination - a single query to update all the exports